snappy/services.go

```diff
@@ -46,6 +51,14 @@ type interacter interface {
 // wait this time between TERM and KILL
 var killWait = 5 * time.Second
 
+const (
+	// the default target for systemd units that we generate
+	servicesSystemdTarget = "multi-user.target"
+
+	// the default target for systemd units that we generate
+	socketsSystemdTarget = "sockets.target"
+)
+
```

Comment:
I would move at least these two constants back to systemd  (I have these mixed feelings about having some form of GenUnitFile there with a template with the common, dependency handling bits in it but may well be overkill)